src/main/java/cst438/Movie.java

```diff
@@ -0,0 +1,92 @@
+package cst438;
+//Domain object
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+//Entity will persist in database
+@Entity 
+
+public class Movie {
+	@Id
+	@GeneratedValue
+	private long id;
+	
+	//Instance variables
+		@NotNull
+		@Size(min=3, max=25)
+		private String userName;
+		
+		@NotNull
+		@Size(min=3, max=25)
+		private String movieTitle;
+		
+		@NotNull
+		@Min(value = 1)
+		@Max(value = 5)
+		private int movieRating;
+		
+		private String date;
```

Comment:
Probably better practice to use java.sql.Timestamp object rather than a raw string